docs/model_developers/release.rst

```diff
@@ -57,6 +61,23 @@ Or for the current project
 
 	inmanta project freeze --recursive --operator ==
 
+Distributing Inmanta modules
+############################
+
+Inmanta modules can be distributed in two different way: using a Git repository or using a Python package. The Python package
+distribution format is supported by V2 modules only.
+
+Git repository distribution format
+----------------------------------
+
+Distributing an Inmanta module using a Git repository happens by storing the source code of that module on a Git repository
+that is accessible by the Inmanta orchestrator. The orchestrator will clone the source code of the module and install it in the
+Inmanta project.
 
-For more information about the ``module.yml`` file see :ref:`module_yml`.
+Python package distribution format
+----------------------------------
 
+Modules defined in the V2 module format can be distributed as a Python package. Run the ``inmanta module package`` command in
```

Comment:
I was going with `inmanta module build` in my PR, though I have no real preference (not a change request per se).

Reply:
I was doubting between those two as well. I don't have a real preference either. I think `build` is probably more in line with the Python terminology. I will update it.